packages/@webex/internal-plugin-feature/src/feature.js

```diff
@@ -43,10 +43,15 @@ const Feature = WebexCore.WebexPlugin.extend({
 
   /**
    * Handles a feature toggle update from the server.
+   * When one of these legacy feature gets updated, this event handler be triggered
+   *     * group-message-notifications
+   *     * mention-notifications
+   *     * thread-notifications,
+   * This handler is being listened from mercury connection success method for the event "event:featureToggle_update"
    * @param {Object} envelope
    * @returns {undefined}
    */
-  handleFeatureUpdate(envelope) {
+  updateFeature(envelope) {
```

Comment:
this is an API of internal-plugin-feature and that plugin shouldn't need to know anything about how the event data coming from mercury is formatted, so I think it would be better if the argument was not `envelope`, because that's a mercury thing, but instead just have the `feature` as the argument

Reply:
Done @marcin-bazyl 

Please check